apps/busybox/cvr_scaleup/test.yml

```diff
@@ -0,0 +1,234 @@
+              shell: kubectl get cvr -n {{ operator_ns }} -l cstorpool.openebs.io/name={{ csp_name.stdout_lines[0] }},cstorvolume.openebs.io/name={{ volume_name.stdout }} -o jsonpath='{.metadata.uid}'
+              register: new_uid
+
+            - name: Creating MD5SUM of new-uid
+              shell: echo {{ new_uid.stdout }} | md5sum | awk '{print $1}' | tr "[a-z]" "[A-z]"
+              register: md5sum
+
+            - name: Replacing the md5sum value in patch yml
+              replace:
+                path: "./patch_cvr.yml"
+                regexp: "<md5-sum-uid>"
+                replace: "{{ md5sum.stdout }}"
+
+            - name: Pathing the cvr's replica id with md5sum
+              shell: kubectl patch cvr {{ volume_name.stdout }}-{{ csp_name.stdout_lines[0] }} -n {{ operator_ns }} --type merge --patch "$(cat patch_cvr.yml)"
+              register: patch_cvr_result
+              failed_when: "'patched' not in patch_cvr_result.stdout"
+            
+            - name: Getting the cstor pool pod name
+              shell: kubectl get pod -n {{ operator_ns }} -l app=cstor-pool,openebs.io/cstor-pool={{ csp_name.stdout_lines[0] }} -o jsonpath='{.items[0].metadata.name}'
+              register: pool_pod_name
+
+            - name: Restarting the pool pod
+              shell: kubectl delete pod {{ pool_pod_name.stdout }} -n {{ operator_ns }}
+
+            - name: Checking for the pool pod to be in running state
+              shell: kubectl get pod -n {{ operator_ns }} -l app=cstor-pool,openebs.io/cstor-pool={{ csp_name.stdout_lines[0] }} -o jsonpath='{.items[0].status.phase}'
+              register: pod_status
+              until: "'Running' in pod_status.stdout"
+              delay: 5
+              retries: 10
+
+            - name: Verify cvr health status
+              shell: kubectl get cvr -n openebs -l openebs.io/persistent-volume={{ volume_name.stdout }} -o jsonpath='{.items[*].status.phase}'
+              register: final_status
+              until: "((final_status.stdout.split()|unique)|length) == 1 and 'Healthy' in final_status.stdout"
+              delay: 3
+              retries: 15
+
+            - name: Verify data persistancy
+              include_tasks: "/utils/scm/applications/busybox/busybox_data_persistence.yml"
+              vars:
+                status: 'VERIFY'
+                ns: "{{ app_ns }}"
+                pod_name: "{{ app_pod_name.stdout }}"
+                testfile: "busybox_data.txt"
+
+            - set_fact:
+                flag: "Pass"
+
+          rescue:
+            - name: Setting fail flag
+              set_fact:
+                flag: "Fail"  
+
+          always:
+        ## RECORD END-OF-TEST IN LITMUS RESULT CR
+            - include_tasks: /utils/fcm/update_litmus_result_resource.yml
+              vars:
+                status: 'EOT'
+
+            - name: Deprovisioning test related resources
+              shell: kubectl delete ns {{ app_ns }}
```

Comment:
Deleting application would be the correct approach @shashank855 rather than deleting namespace